src/character.h

```diff
@@ -150,6 +150,8 @@ enum edible_rating {
     ALLERGY_WEAK,
     // Cannibalism (unless psycho/cannibal)
     CANNIBALISM,
+    // Cannibalism of near human (unless strict humanitarian)
+    STRICT_HUMANITARIANISM,
```

Comment:
This doesn't seem to be used anywhere, or am i missing something?

Reply:
Thank you for looking at it. I'm trying to make a sort of cannibalism lite where most people would consider eating elves or centaurs (in magiclysm), uplifts (in aftershock) and heavily mutated experiments (mainline) to be cannibalism, however for people who with the trait `Strict Humanitarian` they would view elves or uplifted animals as potential food. So I probably need to add that flag to `demihuman meat` if I haven't yet.